models/repo/repo.go

```diff
@@ -196,6 +199,10 @@ func (repo *Repository) SanitizedOriginalURL() string {
 	return u.String()
 }
 
+func (repo *Repository) SizeDetailsString() string {
```

Comment:
How about return an array of struct? We may have AttachmentSize, PackageSize in future.

Reply:
wonder the meaning of `an array of struct`? the `Repository` is already be a struct, add this new function just to avoid too much logic in template file.

Reply:
looks need use something like `ref-issue` if need show many items in feature ...

![image](https://github.com/go-gitea/gitea/assets/25342410/15dcdc5f-e6c1-4533-ab82-98090fcc515f)

Reply:
> wonder the meaning of `an array of struct`? the `Repository` is already be a struct, add this new function just to avoid too much logic in template file.

```go
type SizeDetail struct {
    Name string
    Size int64
}

func (repo *Repository) SizeDetailsString()[]SizeDetail {
```

Reply:
@a1012112796 hi, did a pull request on the requested by @lunny changes please see if ok.
https://github.com/a1012112796/gitea/pull/8

Reply:
@a1012112796 thank you!

Reply:
@lunny so any chances this can get merged, you think?